### PR TITLE
Make hacking lockers require netpass_security, give more ways of finding netpass_security

### PR DESCRIPTION
**Pull request by glassofmilk**
[BALANCE][FEATURE][INPUT WANTED]

## About the PR 
This PR makes three main changes:
1. Using packets to hack lockers will require netpass_security, not netpass_heads
2. Netpass_security is currently only available on the captain's disk. Now it is also available on the disk security members start with, and it's transmitted by the armory authorization computer
3. The armory authorization computer can be hacked using netpass_heads. You'll need to be within 3 tiles to do so, and it will also make a stationwide announcement, so be ready to make a hasty retreat if you plan to do this!

I think the biggest issue with this patch is that the packet emitted by the armory auth computer doesn't really do anything other than provide the netpass_security. This could probably be tied into the receive_signal triggers for lockers and doors, but there's already an area check that would be kind of awkward to replicate, and also I'm lazy. If someone else wants to add more packet integration into this PR, I would welcome it, but I think it could be acceptable without it?

## Why's this needed? 
Currently, hacking lockers open is trivial. This will make it more difficult while still providing multiple methods of doing so.

## Changelog

```
(u)glassofmilk:
(*)Using packets to force open lockers now requires security authorization!
```

### CODE REVIEW

**Comment by Tarmunora** [comment]
remove the easy buddyrecall headpass 2day

**Comment by glassofmilk** [comment]
The buddy recall should probably provide netpass_research, which currently doesn't exist. I think that every department should have its own netpass with different ways of sniffing, and that netpass should be required to open doors and lockers in that department. This is maybe a step in that direction, but I can't do it all at once, and probably not by myself.

**Comment by ThePotato97** [comment]
> 
> 
> The buddy recall should probably provide netpass_research, which currently doesn't exist. I think that every department should have its own netpass with different ways of sniffing, and that netpass should be required to open doors and lockers in that department. This is maybe a step in that direction, but I can't do it all at once, and probably not by myself.

Each head ID could transmit their department's netpass when they open airlocks

**Comment by glassofmilk** [comment]
The transmission range would be short - I think doors currently have a limit of 5 or 8 tiles, something like that. You'd have to be skulking around or tailing someone to sniff their access code. I still think it would be cool if each department had its own special way of sniffing out the access code, but it might be harder to figure out methods for some departments than others.

File: code/datums/jobs.dm [hunk 1]
```diff
@@ -19,6 +19,7 @@
 	var/recieves_miranda = 0
 	var/recieves_implant = null //Will be a path.
 	var/receives_disk = 0
+	var/receives_security_disk = 0
```
Comment (Tarmunora) [inline code/datums/jobs.dm:22]: unused?

Comment (glassofmilk) [reply code/datums/jobs.dm:22]: Oops, forgot to actually check this flag when adding the SECAUTH file to the disk! It's fixed now.

**Comment by zjdtmkhzt** [comment]
Honestly, I feel like it would make the most sense if each department's locker type needed the department's code.

**Pull request closed without merging**